developer/src/kmcmplib/src/CompilerErrors.cpp

```diff
@@ -22,8 +24,8 @@ void ReportCompilerMessage(KMX_DWORD msg, const std::vector<std::string>& parame
   message.lineNumber = kmcmp::currentLine + 1;
   message.columnNumber = kmcmp::ErrChr;
   message.parameters = parameters;
-  //TODO: message.filename =
-  (*kmcmp::msgproc)(message, msgprocContext);
+  message.filename = kmcmp::messageFilename;
+  (*kmcmp::msgproc)(message, kmcmp::msgprocContext);
```

Comment:
Do we have to check that `kmcmp::msgproc != nullptr` ?

Reply:
Good question. This is done in CompilerInterfaces.cpp:

https://github.com/keymanapp/keyman/blob/897697d208880ca216606ae320fc3f504ffedd53/developer/src/kmcmplib/src/CompilerInterfaces.cpp#L27-L30